coccinelle/glnx-errors.cocci

```diff
@@ -0,0 +1,6 @@
+@@
+expression p;
+@@
+- glnx_set_error_from_errno (p);
+- goto out;
++ return  glnx_throw_errno (p);
```

Comment:
(What that spatch *won't* do is rewrite `gboolean ret = FALSE` and the `out` label etc.  That may be possible, but I'd have to learn more spatch)